.env.production

```diff
@@ -0,0 +1,9 @@
+NEXT_PUBLIC_ETH_CHAIN_ID = 1
```

Comment:
Why are we having a prefix of "NEXT_PUBLIC_"?

Reply:
This helps in getting the variable in the browser. If we do not use this prefix then the variable will only be loaded in the Node Js environment and will not be a part of the browser thus will return 'undefined'